src/utils/docker/__tests__/getOptsWithLabels.js

```diff
@@ -0,0 +1,34 @@
+const { getOptsWithLabels } = require('../getOptsWithLabels')
```

Comment:
Not sure if you knew about this, but I wrote this `testEnum` method a while back that makes testing a lot easier in my opinion and it seems it would work well here.
Take a look at the tests for `getImgNameContext.js` => https://github.com/simpleviewinc/keg-cli/blob/master/src/utils/getters/__tests__/getImgNameContext.js

For repetitive tests like this one, it makes things a lot cleaner.

I'm not saying you need to update it, I'm just letting you know incase you missed it.